### PR TITLE
added: carts page

### PR DESCRIPTION
**Pull request by rkapoor10**
carts page

### CODE REVIEW

**Review by Avin008 — COMMENTED** [review]
Everything looks awesome. good to merge

File: ecstylesheets/base.css [hunk 1]
```diff
@@ -31,6 +31,14 @@
     border-bottom: 1px solid #494b4c;
 }
 
+.seperator-lightgray{
+    border-bottom: 1px solid lightgray;
+}
+
+.seperator-dotted{
+    border-bottom: 1px dotted #494b4c;
```
Comment (t007rushi) [inline ecstylesheets/base.css:39]: use color var

File: ecstylesheets/base.css [hunk 2]
```diff
@@ -55,3 +63,33 @@ input[type="radio"] {
...
+.card-container-horizontal{
+    /* width: 680px;
+    height: 250px; */
+    border: none;
+    border-radius: 0rem;
+}
+
+/* .image-container-horizontal{
+    width: 200px;
+    height: 220px;
```
Comment (t007rushi) [inline ecstylesheets/base.css]: same as above

File: ecstylesheets/base.css [hunk 3]
```diff
@@ -55,3 +63,33 @@ input[type="radio"] {
 }
 
 
+.card-container-horizontal{
+    /* width: 680px;
+    height: 250px; */
```
Comment (t007rushi) [inline ecstylesheets/base.css]: remove comments

File: cart-pages/cart.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    align-items: center;
+    padding: 1rem;
+}
+
+.cart-pg-footer{
+    grid-area: foot;
+}
+
+
+
```
Comment (Priya-Patel31) [inline cart-pages/cart.css:33]: remove these spaces .

File: cart-pages/cart.html [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+                <span class="darkteal txt-s alignitems-c"><i class="fas fa-map-marker-alt"></i> Deliver To</span>
+            </div>
+
+
+
+            <div class="cart-products flex-column-center">
+
+                <div class="card-container-horizontal">
+                    <div class="flex-txt">
+                        <img class="image-container-horizontal" src="https://picsum.photos/111/148" alt="">
```
Comment (Priya-Patel31) [inline cart-pages/cart.html]: add something in this alt attribute

**Review by t007rushi — APPROVED** [review]
Do changes Everything looks Clean code

File: ecstylesheets/base.css [hunk 6]
```diff
@@ -55,3 +63,33 @@ input[type="radio"] {
...
+    border: none;
+    border-radius: 0rem;
+}
+
+/* .image-container-horizontal{
```
Comment (Priya-Patel31) [inline ecstylesheets/base.css]: remove these commented code

File: cart-pages/cart.css [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    row-gap: 10px;
+}
+
+
+/* checkout */
+
+.billing-card{
+    background-color: var(--white-color);
+    padding: 2rem;
+    display: flex;
```
Comment (Priya-Patel31) [inline cart-pages/cart.css:45]: use utility classes.

**Review by Priya-Patel31 — APPROVED** [review]
Everything looks good you can merge it !!

File: cart-pages/cart.css (hunk 7 above)
Comment (rkapoor10) [reply cart-pages/cart.css:45]: ok

File: ecstylesheets/base.css (hunk 6 above)
Comment (rkapoor10) [reply ecstylesheets/base.css]: done

File: cart-pages/cart.html (hunk 5 above)
Comment (rkapoor10) [reply cart-pages/cart.html]: done

File: cart-pages/cart.css (hunk 4 above)
Comment (rkapoor10) [reply cart-pages/cart.css:33]: done

**Pull request merged**